specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1527,20 +1527,20 @@
             "$ref": "#/parameters/RelationName"
           },
           {
-            "$ref": "#/parameters/RelationInputModel"
+            "$ref": "#/parameters/Relation"
           }
         ],
         "responses": {
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/BookmarkRelation"
+              "$ref": "#/definitions/Relation"
             }
           },
           "201": {
             "description": "Created",
             "schema": {
-              "$ref": "#/definitions/BookmarkRelation"
```

Comment:
BookmarkRelation and BookmarkRelationList aren't needed in the definition list anymore.